src/lib.rs

```diff
@@ -40,7 +40,7 @@
 //! ```
 //! *(See also [feature flag reference][_features])*
 //!
-//! Then in `main.rs`:
+//! Then in `main.rs`, declare what data you want to parse from the command line by defining a struct and annotating it with clap's derive macros:
```

Comment:
Not liking the flow for this and it feels too verbose for where this is at in the docs.  5e02445c takes another approach

Reply:
sure, that works too